named_entity_recognition/documentation/step-1-setup.md

```diff
@@ -0,0 +1,26 @@
+# Setup
+
+## Deploying Kubeflow to Google Cloud Platform
+This example requires a running Kubeflow environment. The easiest way to setup a Kubeflow environment is by using the [Deployment UI](https://www.kubeflow.org/docs/gke/deploy/deploy-ui/).
```

Comment:
Add the KF version (v0.5.1?)

Reply:
✓ done (added kubeflow version v0.5.0, version currently listed in the deployment UI)